.github/workflows/kotsadm.yaml

```diff
@@ -1,22 +1,40 @@
 name: kotsadm-build
 
 on:
-  push:
-    branches:
-      - "**"
-    tags:
-      - "v*.*.*"
```

Comment:
Does this still work on tags?  This needs to run on when a version tag is created on any branch.

Reply:
removed changes from kotsadm.yaml